projects/media-viewer/src/lib/toolbar/icp-toolbar/icp-toolbar.component.html

```diff
@@ -1,47 +1,47 @@
-<div id="toolbarContainer" class="icpMode">
-  <div id="toolbarViewerLeft">
-    <div class="splitToolbarButton">
-      <button class="toolbarButton info">
-        <span data-l10n-id="icpInfo_label">You are in presentation mode</span>
-      </button>
+<div id="icp-toolbar" class="icpMode">
+  <div id="toolbar-title">
+    <label class="govuk-label" data-l10n-id="icp_label">In court presentation</label>
+  </div>
+
+  <div id="presentation-mode">
+    <div class="item-label">
+      <label class="govuk-label" data-l10n-id="icpInfo_label">You are in presentation mode</label>
     </div>
-    <div class="splitToolbarButton">
-      <button class="toolbarButton" tabindex="0" title="Leave Presentation"
-              (click)="leaveIcpSession()">
-        <span data-l10n-id="icp-leave-presentation_label">Leave Presentation</span>
+    <div class="item-button">
+      <button class="govuk-button govuk-button--secondary" data-module="govuk-button" tabindex="0"
+        title="Leave presentation" (click)="leaveIcpSession()">
+        <span data-l10n-id="icp-leave-presentation_label">Leave presentation</span>
       </button>
     </div>
   </div>
-  <div id="toolbarViewerMiddle">
-    <div class="splitToolbarButton">
-      <button class="toolbarButton info">
-        <span *ngIf="!presenterName" data-l10n-id="noPresenter_label">Waiting for presenter</span>
-        <span *ngIf="presenterName && isPresenter" data-l10n-id="isPresenter_label">You are presenting</span>
-        <span *ngIf="presenterName && !isPresenter"
-              data-l10n-id="otherPresenter_label">{{ presenterName | titlecase }} is presenting</span>
-      </button>
+
+  <div id="presenter-info">
+    <div class="item-label">
+      <label class="govuk-label" *ngIf="!presenterName" data-l10n-id="noPresenter_label">Waiting for presenter</label>
+      <label class="govuk-label" *ngIf="presenterName && isPresenter" data-l10n-id="isPresenter_label">You are
+        presenting</label>
+      <label class="govuk-label" *ngIf="presenterName && !isPresenter"
+        data-l10n-id="otherPresenter_label">{{ presenterName | titlecase }}
+        is presenting</label>
     </div>
-    <div class="splitToolbarButton">
-      <button *ngIf="!presenterName"
-              class="toolbarButton present" id="icp-present" tabindex="0" title="Present"
-              (click)="present()">
-        <span data-l10n-id="icp-present_label">Present</span>
+    <div class="item-button">
+      <button *ngIf="!presenterName" class="govuk-button" id="icp-present" tabindex="0" title="Present"
+        (click)="present()">
+        <span data-l10n-id="icp-present_label">Start presenting</span>
```

Comment:
the labels came from the product manager, did you change it based on user/business feedback?

Reply:
The label change is part of a sub-task for the ICP toolbar amends (https://tools.hmcts.net/jira/browse/EUI-2495). Assuming this is correct the corresponding l10n localization phrase would also need to change.